src/smt/optimization_solver.h

```diff
@@ -221,7 +219,7 @@ class OptimizationSolver
   /**
    * Constructor
    * @param parent the smt_solver that the user added their assertions to
-   **/
+   */
```

Comment:
It would be better not to include changes to the comments of unrelated methods in this PR.

Reply:
oh I really don't want to add them back...